app/views/diary_entries/index.html.erb

```diff
@@ -14,15 +14,19 @@
         <ul class="clearfix">
           <% unless params[:friends] or params[:nearby] -%>
             <li><%= rss_link_to :action => "rss", :language => params[:language] %></li>
-            <% end -%>
+          <% end -%>
 
-            <% if @user && @user == current_user || !@user && current_user %>
-              <li><%= link_to image_tag("new.png", :class => "small_icon") + t(".new"), new_diary_entry_path, :title => t(".new_title") %></li>
-            <% end %>
+          <% @languages&.each do |language| %>
```

Comment:
I think these should probably go after "New Diary Entry" and before "My Diary" they're links to a subset of the diary in much the same way that "My Diary" is so they should probably be together.

Reply:
What I originally wanted to do is to move the "New Diary Entry" to the end, but I decided not to do it in the same pull request.

I moved it now:

![image](https://github.com/openstreetmap/openstreetmap-website/assets/4158490/719155aa-32b7-4f21-b90d-4d6c8d79252e)